packages/n4s/src/rules/__tests__/isArrayOf.test.js

```diff
@@ -0,0 +1,40 @@
+import { isArrayOf } from 'isArrayOf';
+import enforse from 'enforce'
+
+describe('Tests isArrayOf rule', () => {
+
+  it('Should return true if all elements are ture for one or more rules', () => {
+    expect(isArrayOf([3,4,5,'six',7],
+     enforse.greaterThan(2), 
+     enforse.isString())
+     ).toBe(true);
+  });
+
+
+  it('Should return false if one element or more fails all rules', () => {
+    expect(isArrayOf([3,4,5,['s','i','x'],7], 
+    enforse.greaterThan(2), 
+    enforse.isString())
+    ).toBe(false);
+  });
+
+
+  describe('Tests for recursive call', ()=>{
+
+    it('Should return true if all elements are ture for one or more rules', () => {
```

Comment:
typo 🙈
```suggestion
    it('Should return true if all elements are true for one or more rules', () => {
```


also here: 
https://github.com/ealush/vest/pull/499/files#diff-e2dfd7aa604cef61b1c17ca8cf0a9e4fc5bbe176d63350fbf4a01a6d95f68219R6